### PR TITLE
Responsive styling

### LINKED ISSUE #53: Media queries!!

### PR DESCRIPTION
**Pull request by novaraptur**
# Description

Added media queries to nav bar styling, featured movie section styling, and movie cards styling.

Fixes #53 

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] Refactor

# How Has This Been Tested?

- [X] Console Dev Tools

### CODE REVIEW

File: src/components/Movies/Movies.css [hunk 1]
```diff
@@ -3,3 +3,15 @@
...
   padding: 1em;
 }
+
+@media only screen and (max-width: 900px) {
+  .movies-container {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+}
+
+@media only screen and (max-width: 600px) {
```
Comment (clairefields15) [inline src/components/Movies/Movies.css:13]: can we add one more query for iphone size that has one column? so then 2 columns is for like a small tablet size and 3 is for large tablet/small laptop

**Review by clairefields15 — CHANGES_REQUESTED** [review]
Great job! I would love one more query for the smallest screen that has just one column

File: src/components/Movies/Movies.css (hunk 1 above)
Comment (novaraptur) [reply src/components/Movies/Movies.css]: Just updated this!

Comment (clairefields15) [reply src/components/Movies/Movies.css]: Awesome!

**Pull request merged**